core/src/main/java/org/commonjava/indy/core/expire/TimeoutEventListener.java

```diff
@@ -189,7 +189,7 @@ public void onFileAccessEvent( @Observes final FileAccessEvent event )
                 }
                 else if ( type == StoreType.remote )
                 {
-                    SpecialPathInfo info = specialPathManager.getSpecialPathInfo( transfer );
+                    SpecialPathInfo info = specialPathManager.getSpecialPathInfo( transfer, key.getPackageType() );
```

Comment:
do you mean this lien fix the real problem? in another words, if no packagetype, the npm info not tell the truth. am I right?

Reply:
Seems so, the default value of this parameter is "maven".

Reply:
Cool. LGTM